src/main/java/org/kainos/ea/api/ProjectService.java

```diff
@@ -0,0 +1,31 @@
+package org.kainos.ea.api;
+
+import org.kainos.ea.cli.ProjectRequestAddClient;
+import org.kainos.ea.client.FailedToUpdateProjectException;
+import org.kainos.ea.client.InvalidProjectException;
+import org.kainos.ea.core.ProjectValidator;
+import org.kainos.ea.db.ProjectDao;
+
+import java.sql.SQLException;
+
+public class ProjectService {
+    private ProjectDao projectDao = new ProjectDao();
+    private ProjectValidator projectValidator = new ProjectValidator();
```

Comment:
pass these objects in via a constructor (ask Callum, he's an expert)

Reply:
done